### ISSUE TITLE
Create release with Scala version > 2.13.12

### ISSUE THREAD

**Issue by astamagnusdottir-asana** [scalapb/protoc-bridge#406]
The current release `0.9.7` uses Scala version `2.13.12`, but there is an issue in the `rules_scala` [package](https://github.com/bazelbuild/rules_scala) where the Reporter classes are not minor version compatible between `2.13.12` and `2.13.13`. This causes an error during compilations for projects building with a Scala version > 2.13.12.

Error from build:
```
external/io_bazel_rules_scala/src/java/io/bazel/rulesscala/scalac/deps_tracking_reporter/after_2_12_13_and_before_2_13_12/DepsTrackingReporter.java:235: error: method warning in class Reporter cannot be applied to given types;
            reporter.warning(NoPosition$.MODULE$, strictDepsReport.toString());
                    ^
  required: Position,String,List<CodeAction>
  found:    NoPosition$,String
  reason: actual and formal argument lists differ in length
```

The only external library we use which is not on `version >= 2.13.13` is the `protoc-bridge`.

The code is already in place in the repository as in the `build.sbt` on the master branch builds agains `2.13.15`. Would it be possible to create a release or a patch release with the new version dependencies to support this minor version incompatibility?

cc: @thesamet

**Comment by thesamet** [comment]
I released 0.9.8 today.

**Comment by astamagnusdottir-asana** [comment]
Excellent, thank you!
